libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/internal/MapboxTripNotification.kt

```diff
@@ -159,6 +160,13 @@ class MapboxTripNotification constructor(
     override fun onTripSessionStarted() {
         registerReceiver()
         notificationActionButtonChannel = Channel(1)
+
+        collapsedNotificationRemoteViews?.apply {
+            setViewVisibility(R.id.navigationIsStarting, View.VISIBLE)
+        }
+        expandedNotificationRemoteViews?.apply {
+            setViewVisibility(R.id.navigationIsStarting, View.VISIBLE)
+        }
```

Comment:
the same lines of code in `updateNotificationViews `

move to a separate fun with `visibility` param?

Reply:
I think `MapboxTripNotification.kt` has a lot of things that should be refactored(like here 2 almost the same views which might be hidden under interfaces, also we can migrate to kotlin), so let's keep it as is, I created an issue for class refactoring https://github.com/mapbox/mapbox-navigation-android/issues/4163